sentinel-extension/sentinel-annotation-aspectj/README.md

```diff
@@ -13,8 +13,11 @@ The `@SentinelResource` annotation indicates a resource definition, including:
   - The return type should match the origin method;
   - The parameter list should match the origin method, and an additional `Throwable` parameter can be provided to get the actual exception.
 - `defaultFallback` (since 1.6.0): The default fallback method when exceptions caught (including `BlockException`, but except the exceptions defined in `exceptionsToIgnore`). Its intended to be a universal common fallback method. The method should be located in the same class with original method by default. If you want to use method in other classes, you can set the `fallbackClass` with corresponding `Class` (Note the method in other classes must be *static*). The default fallback method signature requirement:
+  - supported class-level annotation (since 1.8.0);
   - The return type should match the origin method;
   - parameter list should be empty, and an additional `Throwable` parameter can be provided to get the actual exception.
+- `fallbackClass` The default fallback method in which class.
+  - supported class-level annotation (since 1.8.0), Effective when the class annotation `defaultFallback` is not empty;
```

Comment:
Duplicate with the content above:

> If you want to use method in other classes, you can set the `fallbackClass` with corresponding `Class` (Note the method in other classes must be *static*).

And it's effective for both `fallback` and `defaultFallback`.